src/d3a/models/area/stats.py

```diff
@@ -50,16 +50,26 @@ def __init__(self, area_markets):
     def update_aggregated_stats(self, area_stats):
         self.aggregated_stats = area_stats
 
+    def _extract_from_bills(self, trade_key):
+        if self.current_market is None:
+            return None
+        return {key: self.aggregated_stats["bills"][trade_key][key]
+                for key in ["earned", "spent", "bought", "sold"]} \
+            if "bills" in self.aggregated_stats \
+               and trade_key in self.aggregated_stats["bills"] else None
```

Comment:
```suggestion
               and trade_key in self.aggregated_stats["bills"] else {}
```
looks better to return an empty dictionary than `None` to have a uniform data type pattern  irrespective of if value is there or not.